static/js/components/OrderSummary_test.js

```diff
@@ -64,26 +64,35 @@ describe('OrderSummary', () => {
     );
   };
 
-  it('shows discount calculation if user has a coupon', () => {
-    let course = findCourse(course => (
-      course.runs.length > 0 &&
-      course.runs[0].status === STATUS_OFFERED
-    ));
-    let firstRun = course.runs[0];
-    const wrapper = renderOrderSummary({
-      courseRun: firstRun,
-      course: course,
-      discount: COURSE_PRICES_RESPONSE[1].price,
-      finalPrice: 0,
-      couponCode: '561KH'
+  [
+    ['561KH', 'non-blank coupon code'],
+    [null, 'null coupon code'],
+    ['', 'blank coupon code']
+  ].forEach(([code, codeDescription]) => {
+    it(`shows discount calculation if user has a coupon with code ${codeDescription}`, () => {
```

Comment:
one last nitpick: please change this to `shows discount calculation if user has a coupon with ${codeDescription}`. the way it is now, an example description would be "shows discount calculation if user has a coupon with code blank coupon code". we want that to be "shows discount calculation if user has a coupon with ~~code~~ blank coupon code"